README.md

```diff
@@ -185,9 +185,39 @@ Check out an example configuration in the [examples/complete](./examples/complet
 
 <!-- NOTE to Masterpoint team: We might want to create a small wrapper to automatize this using Taskit. On hold for now. -->
 
-### What goes in a Stack config file? e.g. `stacks/dev.yaml`, `stacks/common.yaml`, `stack.yaml`, etc
-
-Most settings that you would set on [the Spacelift Stack resource](https://search.opentofu.org/provider/spacelift-io/spacelift/latest/docs/resources/stack) are supported. Additionally, you can include certain Stack specific settings that will override this module's defaults like `default_tf_workspace_enabled`, `tfvars.enabled`, and similar. See the code for full details.
+### What goes in a Stack config file? e.g. `stacks/dev.yaml`, `stacks/common.yaml`, `stack.yaml`, and similar
+
+Most settings that you would set on [the Spacelift Stack resource](https://search.opentofu.org/provider/spacelift-io/spacelift/latest/docs/resources/stack) are supported. Additionally, you can include certain automation settings that will override this module's defaults like `automation_settings.default_tf_workspace_enabled`, `automation_settings.tfvars_enabled`, and similar.
+
+Below is a brief example. You can also see the full schema in our [JSON Schema file](./stack-config.schema.json).
```

Comment:
How to ensure this schema file is up-to-date with the current code base? How a contributor should update it if they update settings?

Reply:
This is a good question @gberenice... I don't know the answer. Let me ask the Cloud Posse folks -- Maybe they have a system?

> Also, I think this should be a major bump. We can keep schema versions corresponding to release major versions.

Agreed. Added a `!` to PR title, which should cause us to bump to v1.0 👍

Reply:
Any news on how to keep the schema up-to-date? In case we don't have a solid approach for now, at least let's drop a note on how to update it and how we're going to manage release versions.

Reply:
I added a test for keeping the schema up-to-date as part of #44 / this commit: https://github.com/masterpointio/terraform-spacelift-automation/pull/44/commits/32b679fcf688f09b1be6488fb066bde9247a3de4

We'll go with that for now and see how it works!